### PR TITLE
Update Managers (data.yml)

### PR DESCRIPTION
**Pull request by Haselnussbomber**
I tried to give things a more uniform name based on newer managers.
Also I added a couple new managers. Most of them don't exist in the old exe so I made up names based on lua functions they are used in or what sheets they interact with. 😅
Hope that is alright?

### CODE REVIEW

File: ida/data.yml [hunk 1]
```diff
@@ -932,11 +932,11 @@ classes:
...
     funcs:
-      0x1412EC2D0: HasValue  # g_FateTablePtr != 0
-      0x1412EC2F0: GetSingleton
-      0x1412EC300: Initialize
-      0x1412ECAD0: ctor
-      0x1412EC350: dtor
+      0x1412EC2D0: HasInstance
+      0x1412EC2F0: GetInstance
+      0x1412EC300: Create
+      0x1412EC350: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:938]: Should be still named `dtor`

File: ida/data.yml [hunk 2]
```diff
@@ -932,11 +932,11 @@ classes:
...
+      0x1412ECAD0: Initialize
```
Comment (wolfcomp) [inline ida/data.yml:939]: Should still be named `ctor`

File: ida/data.yml [hunk 3]
```diff
@@ -963,8 +963,9 @@ classes:
       - ea: 0x1421F6E20
         pointer: False
     funcs:
-      0x140745580: ctor
-      0x1407455D0: dtor
+      0x140746820: ctor
+      0x140745580: Create
+      0x1407455D0: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:968]: Should still be named `dtor`

File: ida/data.yml [hunk 4]
```diff
@@ -963,8 +963,9 @@ classes:
       - ea: 0x1421F6E20
         pointer: False
     funcs:
-      0x140745580: ctor
-      0x1407455D0: dtor
+      0x140746820: ctor
+      0x140745580: Create
```
Comment (wolfcomp) [inline ida/data.yml:967]: Should still be called `ctor`

File: ida/data.yml [hunk 5]
```diff
@@ -963,8 +963,9 @@ classes:
       - ea: 0x1421F6E20
         pointer: False
     funcs:
-      0x140745580: ctor
-      0x1407455D0: dtor
+      0x140746820: ctor
```
Comment (wolfcomp) [inline ida/data.yml:966]: Should be called `Initialize`

File: ida/data.yml [hunk 6]
```diff
@@ -991,6 +992,25 @@ classes:
...
+      0x1414244D0: Finalize
+  Client::Game::ServerRequestCallbackManager:
+    instances:
+      - ea: 0x1422382A8
+        pointer: True
+    funcs:
+      0x141424530: Initialize
+      0x1414245E0: HasInstance
+      0x141424600: GetInstance
+      0x141424610: Create
```
Comment (wolfcomp) [inline ida/data.yml:1011]: Should be called `ctor`

File: ida/data.yml [hunk 7]
```diff
@@ -991,6 +992,25 @@ classes:
...
+      0x141424660: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1012]: Should be called `dtor`

File: ida/data.yml [hunk 8]
```diff
@@ -1073,8 +1094,8 @@ classes:
       - ea: 0x1421FF168
         pointer: True
     funcs:
-      0x1408A4B00: ctor
-      0x1408A4B50: dtor
+      0x1408A4B00: Create
```
Comment (wolfcomp) [inline ida/data.yml:1097]: Should still be named `ctor`

File: ida/data.yml [hunk 9]
```diff
@@ -1073,8 +1094,8 @@ classes:
...
+      0x1408A4B50: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1098]: Should still be named `dtor`

File: ida/data.yml [hunk 10]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
       0x140B1D440: GetWeatherForHour
       0x140B1D620: GetWeatherForDaytime
+  Client::Game::FashionCheckManager: # Fashion Report
+    instances:
+      - ea: 0x1421FF160
+        pointer: True
+    funcs:
+      0x1408A8700: HasInstance
+      0x1408A8720: GetInstance
+      0x1408A85F0: Create
```
Comment (wolfcomp) [inline ida/data.yml:1178]: Should be named `ctor`

File: ida/data.yml [hunk 11]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x1408A86D0: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1179]: Should be named `dtor`

File: ida/data.yml [hunk 12]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+    vtbls:
+      - ea: 0x141B64740
+        base: Client::Game::ServerRequestCallbackInterface
+    instances:
+      - ea: 0x142238B50
+        pointer: True
+    funcs:
+      0x141430480: HasInstance
+      0x141430470: GetInstance
+      0x1414304A0: Create
```
Comment (wolfcomp) [inline ida/data.yml:1190]: Should be named `ctor`

File: ida/data.yml [hunk 13]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x141430500: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1191]: Should be named `dtor`

File: ida/data.yml [hunk 14]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::QuestRecompleteManager: # Seasonal Event Replay
+    instances:
+      - ea: 0x142238B40
+        pointer: True
+    funcs:
+      0x14142FEB0: Initialize
+      0x14142FF40: Finalize
+      0x14142FFF0: HasInstance
+      0x14142FFE0: GetInstance
+      0x141430010: Create
```
Comment (wolfcomp) [inline ida/data.yml:1201]: Should be named `ctor`

File: ida/data.yml [hunk 15]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x141430060: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1202]: Should be named `dtor`

File: ida/data.yml [hunk 16]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::QuestRedoManager: # New Game+
+    instances:
+      - ea: 0x14222AEF8
+        pointer: True
+    funcs:
+      0x140B901B0: Initialize
+      0x140B90320: Finalize
+      0x140B90460: HasInstance
+      0x140B90480: GetInstance
+      0x140B90490: Create
```
Comment (wolfcomp) [inline ida/data.yml:1212]: Should be named `ctor`

File: ida/data.yml [hunk 17]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x140B904E0: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1213]: Should be named `dtor`

File: ida/data.yml [hunk 18]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::QuestRedoMapMarkerManager:
+    instances:
+      - ea: 0x142238B60
+        pointer: True
+    funcs:
+      0x141430B70: Initialize
+      0x141433010: Finalize
+      0x141430C20: HasInstance
+      0x141430C10: GetInstance
+      0x141430C40: Create
```
Comment (wolfcomp) [inline ida/data.yml:1223]: Should be named `ctor`

File: ida/data.yml [hunk 19]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x141430C90: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1224]: Should be named `dtor`

File: ida/data.yml [hunk 20]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::FreeMoveManager:
+    instances:
+      - ea: 0x142238B28
+        pointer: True
+    funcs:
+      0x14142C870: Initialize
+      0x14142C940: Finalize
+      0x14142CB30: HasInstance
+      0x14142CB20: GetInstance
+      0x14142CB50: Create
```
Comment (wolfcomp) [inline ida/data.yml:1234]: Should be named `ctor`

File: ida/data.yml [hunk 21]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x14142CBA0: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1235]: Should be named `dtor`

File: ida/data.yml [hunk 22]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::SequentialEventManager:
+    instances:
+      - ea: 0x142238B70
+        pointer: True
+    funcs:
+      0x1414336F0: Initialize
+      0x141433A90: Finalize
+      0x141433D70: HasInstance
+      0x141433D60: GetInstance
+      0x141433D90: Create
```
Comment (wolfcomp) [inline ida/data.yml:1245]: Should be named `ctor`

File: ida/data.yml [hunk 23]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x141433DE0: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1246]: Should be named `dtor`

File: ida/data.yml [hunk 24]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::PathWalkManager:
+    instances:
+      - ea: 0x142238B80
+        pointer: True
+    funcs:
+      0x1414386B0: Initialize
+      0x141438B00: Finalize
+      0x141438750: HasInstance
+      0x141438740: GetInstance
+      0x141438770: Create
```
Comment (wolfcomp) [inline ida/data.yml:1256]: Should be named `ctor`

File: ida/data.yml [hunk 25]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x1414387C0: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1257]: Should be named `dtor`

File: ida/data.yml [hunk 26]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::QuestEffectManager:
+    instances:
+      - ea: 0x142238B18
+        pointer: True
+    funcs:
+      0x14142B020: Initialize
+      0x14142B680: Finalize
+      0x14142B5D0: HasInstance
+      0x14142B5C0: GetInstance
+      0x14142B5F0: Create
```
Comment (wolfcomp) [inline ida/data.yml:1267]: Should be named `ctor`

File: ida/data.yml [hunk 27]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x14142B640: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1268]: Should be named `dtor`

File: ida/data.yml [hunk 28]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::DawnManager:
+    instances:
+      - ea: 0x1421FF158
+        pointer: True
+    funcs:
+      0x1407C82B0: Initialize
+      0x1407C7A70: HasInstance
+      0x1407C7A90: GetInstance
+      0x1407C79F0: Create
```
Comment (wolfcomp) [inline ida/data.yml:1277]: Should be named `ctor`

File: ida/data.yml [hunk 29]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x1407C7A40: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1278]: Should be named `dtor`

File: ida/data.yml [hunk 30]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+  Client::Game::ServerValueCallbackManager:
+    instances:
+      - ea: 0x142238EE0
+        pointer: True
+    funcs:
+      0x14143AFF0: HasInstance
+      0x14143AFE0: GetInstance
+      0x14143B010: Create
```
Comment (wolfcomp) [inline ida/data.yml:1286]: Should be named `ctor`

File: ida/data.yml [hunk 31]
```diff
@@ -1147,12 +1168,130 @@ classes:
...
+      0x14143B060: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:1287]: Should be named `dtor`

File: ida/data.yml [hunk 32]
```diff
@@ -10894,18 +11034,19 @@ classes:
       - ea: 0x1421FF140
         pointer: False
     funcs:
-      0x1408E8660: ctor
-      0x1408E8710: dtor
+      0x1408E8660: Create
```
Comment (wolfcomp) [inline ida/data.yml:11037]: Should still be named `ctor`

File: ida/data.yml [hunk 33]
```diff
@@ -10894,18 +11034,19 @@ classes:
...
+      0x1408E8710: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:11038]: Should still be named `dtor`

File: ida/data.yml [hunk 34]
```diff
@@ -10894,18 +11034,19 @@ classes:
...
       0x1408E8790: GetInstance
   Client::Game::Event::EventFramework:
     instances:
       - ea: 0x1421FF148
         pointer: True
     funcs:
-      0x140863BD0: Initialize
-      0x140863C80: GetSingleton
-      0x140863F70: ctor
+      0x140863BD0: Create
```
Comment (wolfcomp) [inline ida/data.yml:11046]: Should be named `ctor`

File: ida/data.yml [hunk 35]
```diff
@@ -10894,18 +11034,19 @@ classes:
...
+      0x140863C30: Destroy
```
Comment (wolfcomp) [inline ida/data.yml:11047]: Should be named `dtor`

**Review by wolfcomp — CHANGES_REQUESTED** [review]
Just a few lines to match what other structs have them named.

If you also want to include `140863C30` is `Client::Game::Event::EventFramework.dtor`

File: ida/data.yml (hunk 13 above)
Comment (Haselnussbomber) [reply ida/data.yml:1191]: I reversed this because the dtor is part of the vtable (@ 0x141B64740).

File: ida/data.yml (hunk 33 above)
Comment (Haselnussbomber) [reply ida/data.yml:11042]: This also is not the dtor. The dtor is part of the vtable (@ 0x141A6A198).

File: ida/data.yml (hunk 2 above)
Comment (Haselnussbomber) [reply ida/data.yml:939]: As we talked about it, the ctor allocates the thing. This address does not do that.

**Review by wolfcomp — APPROVED** [review]

**Pull request merged**